test/jupyter-datascience-notebooks/test.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+cd "$(dirname "$0")" || exit
```

Comment:
Out of curiosity, why is this here? 🤔 consistency across different dirs?

Reply:
I think this script expects to call `test-utils.sh`, so it needs to go to a specific directory.